src/components/Image/Image.jsx

```diff
@@ -0,0 +1,77 @@
+import "../../styles.css";
+import "./img-styles.css";
+
+export const Image = () => {
+  return (
+    <div className="w-full m-auto mb-8 text-left">
+      <h1 className="mb-6 text-3xl">Image</h1>
+      <p className="mb-6 text-lg text-gray-600">
+        To use a responsive image add
+        <code className="code-line">.img</code> class to your{" "}
+        <code className="code-line">
+          {" "}
+          &lt;img src="img/example.jpg" alt="mountain"/&gt;{" "}
+        </code>
+        . The image will adjust its width according to its parent container
+        while maintaing its aspect ratio.
+      </p>
+      {/* normal responsive image */}
+      <div className="content-container">
+        <img
+          src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTGOGbjJ-FuxPywVsQx6Csc8xeRb7JpfDKRAw&usqp=CAU"
+          alt="space"
+          className="img"
```

Comment:
class name can be improved img -> " responsive-img" !! I

Reply:
Yes, it will make more sense that way. I will change it. Thank you.